### ISSUE TITLE
Gradients do not exist for variables ['lstm1/nested_lstm_cell/input_kernel_1:0']

### ISSUE THREAD

**Issue by BogdanM1** [titu1994/Nested-LSTM#5]
With tensorflow version 2.3.0 and keras version 2.4.3 I get the following warning: 
Gradients do not exist for variables ['lstm1/nested_lstm_cell/input_kernel_1:0']  when minimizing the loss.

This is a piece of code I used: 
from nested_lstm import NestedLSTM

i = Input(shape=(10, 4), name='input_layer')
o = NestedLSTM(28, depth=4, return_sequences=True, name='lstm1')(i) 

Any ideas why this happens?

**Comment by denfenser** [comment]
I also fing the warning,and I choose to ingore it ! The warning didn't cause some serious problems.
